editor/src/node_graph_executor.rs

```diff
@@ -524,6 +532,10 @@ impl NodeGraphExecutor {
 
 				//return Err("Graphic group (see console)".to_string());
 			}
+			TaggedValue::Palette(palette) => {
+				debug!("{palette:#?}");
+				return Err("Palette (see console)".to_string());
+			}
```

Comment:
I have printed it following what exists today to Artboard, what just gets the input parameter and prints it on the browser console.